website/docs/reference/front-end-api.md

```diff
@@ -1,5 +1,5 @@
 ---
-title: Front-end API Access
+title: Frontend API
```

Comment:
Applying correct spelling and capitalization (given it's a product name, and needs to be differentiated)